cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -0,0 +1,504 @@
+# -*- coding: utf-8 -*-
+# These tests don't work at the moment, due to the security_groups multi select not working
+# in selenium (the group is selected then immediately reset)
+from textwrap import dedent
+
+import fauxfactory
+import pytest
+from riggerlib import recursive_update
+from widgetastic_patternfly import CheckableBootstrapTreeview as Check_tree
+
+from cfme import test_requirements
+from cfme.cloud.provider import CloudInfraProvider
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider.gce import GCEProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.fixtures.provider import setup_one_or_skip
+from cfme.markers.env_markers.provider import providers
+from cfme.utils import normalize_text
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.providers import ProviderFilter
+from cfme.utils.log import logger
+from cfme.utils.update import update
+from cfme.utils.wait import wait_for
+
+pf = ProviderFilter(classes=[CloudInfraProvider], required_flags=['provision'])
```

Comment:
Not used in an env marker, and duplicated within a_provider below, I think this is unnecessary.